idseq_dag/engine/pipeline_step.py

```diff
@@ -52,9 +52,14 @@ def __init__(self, name, input_files, output_files,
         self.exec_thread = None
         self.upload_thread = None
         self.input_files_local = []
-        self.additional_files_to_upload = []
-        self.optional_files_to_upload = []
-        self.additional_folders_to_upload = []
+
+        # Extra output files for internal use, not visible to users
+        self.additional_output_files_hidden = []
+        # Extra output files available to users
```

Comment:
Can you add that they are uploaded to s3 and exposed in the stage status file in the comments?